src/main/java/me/luligabi/logicates/common/block/logicate/inputless/timer/TimerLogicateBlockEntity.java

```diff
@@ -63,13 +63,16 @@ public static void tick(World world, BlockPos pos, BlockState state, TimerLogica
             blockEntity.ticks++;
         } else {
             world.setBlockState(pos, state.with(LogicateBlock.POWERED, true), 1);
-            world.setBlockState(pos, state.with(LogicateBlock.POWERED, false), 1);
             if(!blockEntity.mute) {
                 world.playSound(null, pos.getX(), pos.getY(), pos.getZ(), SoundEvents.UI_BUTTON_CLICK,
                         SoundCategory.BLOCKS, 0.5F, 1F, pos.asLong());
             }
             blockEntity.ticks = 0;
         }
+
+        if (blockEntity.ticks == 1) {
```

Comment:
```suggestion
        if(blockEntity.ticks == 1) {
```

I feel bad for being this nitpicky, hopefully you should be able to just have Github do the job of commiting this lol